src/plugins/kibana/public/kibana.js

```diff
@@ -11,7 +11,7 @@ const chrome = require('ui/chrome');
 const routes = require('ui/routes');
 const modules = require('ui/modules');
 
-const kibanaLogoUrl = require('ui/images/kibana.svg');
+var kibanaLogoUrl = require('ui/images/bitergia-analytics-logo.png');
```

Comment:
This code comes from Kibana. Probably it was var when we start working on it, then they changed to const. It is better probably to use const.